examples/server-actions-upload/app/action.ts

```diff
@@ -0,0 +1,25 @@
+'use server'
+
+import fs from 'fs'
+import path from 'path'
+import { promisify } from 'util'
+
+const writeFile = promisify(fs.writeFile)
+
+export async function saveData(formData: FormData) {
+  const file = formData.get('file') as File
+  const filePath = path.join('./', file.name)
+
+  const buffer = await file.arrayBuffer()
+
+  await writeFile(filePath, Buffer.from(buffer))
```

Comment:
Why is this writing a file here?

Reply:
> Why is this writing a file here?

Because uploaded file should save to somewhere 

Or I change save the file to  `/tmp` folder?  follow: https://vercel.com/guides/how-can-i-use-files-in-serverless-functions

Reply:
I just change the path to `/tmp` to save upload file.